library/core/src/option.rs

```diff
@@ -1656,7 +1656,8 @@ impl<T, E> Option<Result<T, E>> {
 }
 
 // This is a separate function to reduce the code size of .expect() itself.
-#[inline(never)]
+#[cfg_attr(not(feature = "panic_immediate_abort"), inline(never))]
+#[cfg_attr(feature = "panic_immediate_abort", inline)]
 #[cold]
```

Comment:
Does `#[inline]` have any effect when it's also `#[cold]`?

Reply:
No idea! We could gate that attribute too, I guess